### PR TITLE
[TECH]: update sign-in flow

### PR DESCRIPTION
**Pull request by eliobricenov**
- update account avatar
- add logout 
- remove protocol handler we had for email deeplinking, we use the same that we use for oauth deeplinking 

**Qase test suite: https://app.qase.io/project/HC?suite=31&previewMode=side**

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -8,4 +8,5 @@ export const METAMASK_SNAPS_URL = 'https://snaps.metamask.io/'
 export const METAMASK_SNAPS_EXTERNAL_URL = 'https://metamask.io/snaps/'
 export const METAMASK_PORTFOLIO_URL = 'https://portfolio.metamask.io/'
 
-export const DEV_PORTAL_URL = 'https://developers.hyperplay.xyz/'
+export const DEV_PORTAL_URL =
```
Comment (eliobricenov) [inline src/common/constants.ts:11]: adding note to use production when https://github.com/valist-io/hyperplay-dev/pull/180 is merged

**Review by BrettCleary — APPROVED** [review]
This is all working perfectly for me. Great work! 

I just think we need to update the avatar icon so that it can convey both
1. wallet connected/disconnected
2. hp account signed in/signed out

**Comment by eliobricenov** [comment]
> This is all working perfectly for me. Great work!
> 
> I just think we need to update the avatar icon so that it can convey both
> 
> 1. wallet connected/disconnected
> 2. hp account signed in/signed out

Yup, Pimentel updated those today https://www.figma.com/file/GKPELZxk551oVC3EDLgEDA/Current-Design?type=design&node-id=3956-40397&mode=design&t=DVDlltJpbfYy46Wi-0

I'll work on those after I finish reviews

**Review by nyghtstalker — APPROVED** [review]
Works great! Awesome job!

- Confirmed the MM and WalletConnect icons when the wallets are connected. 
- Confirmed the avatar appears when logged in and does not when logged out.
- Confirmed all the social boxes work and connects to account.

**Pull request merged**